ui/job-view/pushes/PushList.jsx

```diff
@@ -141,6 +139,8 @@ class PushList extends React.Component {
     }
     return (
       <div
+        role="button"
```

Comment:
So, you've handled this case correctly as well.  

Just for some context:  This, and other special cases for clicking, are an attempt to reduce the number of event bindings on objects when we have a lot of shown objects.  We just handle the events on the parent to reduce our memory footprint.  Earlier attempts that had tons of event handler for each object was cumbersome and expensive ( memory-wise ).

Reply:
Interesting, that's good to know.